albums/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.utils import timezone
+import albums
+from artists.models import Artist
+
+# Create your models here.
+class Album(models.Model):
+    name = models.CharField(default='New Album',max_length=255)
+    creation_date = models.DateTimeField(default=timezone.now)
+    release_date =  models.DateTimeField(blank=False)
+    cost = models.DecimalField(blank=False,decimal_places=2,max_digits=15)
+    artist = models.ForeignKey(Artist,on_delete=models.CASCADE)
+    album_approved = models.BooleanField(default=False)
```

Comment:
the naming convention for boolean fields in Django should have the prefix "is_" so here it should be is_approved